dubbo-registry/dubbo-registry-multicast/src/test/java/org/apache/dubbo/registry/multicast/MulticastRegistryTest.java

```diff
@@ -39,7 +39,7 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class MulticastRegistryTest {
```

Comment:
Why change this file

Reply:
Since Junit 5, it supports classes which has package private access, so it's redundant, and I have Sonarlint installed, it complains the modifier...